src/presenter/trip-list-presenter.js

```diff
@@ -0,0 +1,20 @@
+import {render} from '../render.js';
+import TripListView from '../view/trip-list-view.js';
+import EditFormView from '../view/edit-form-view.js';
+import WaypointView from '../view/waypoint-view.js';
```

Comment:
Немного нестандартное название, не настаиваю но как на счет EventView EventPointView
```suggestion
import WayPointView from '../view/way-point-view.js';
```